CONTRIBUTING.md

```diff
@@ -21,14 +21,14 @@ Contributors should follow the following process:
 7. Create a pull request for your changes on the PrestaShop project. Be sure to follow [the contribution guidelines][2] in your pull request. If you need help to make a pull request, read the [Github help page about creating pull requests][3].
 8. Wait for one of the core developers either to include your change in the codebase, or to comment on possible improvements you should make to your code.
 
-That's it: you have contributed to this open source project! Congratulations!
+That's it: you have contributed to this open-source project! Congratulations!
```

Comment:
I don't think standard naming contains a `-`.
See https://opensource.com/resources/what-open-source

Reply:
The first line of this document begins with this sentence: PrestaShop is an open-source e-commerce solution.
 and I wanted all the naming be the same.

Reply:
Haha 😅 I'm quite confident actually it's the 1st sentence that is wrong, the `-` should not be there.

We can fix the 1st sentence in another PR 😉

Reply:
Hi! I confirm, we should write: 
"open source" and "e-commerce" ☺️

Reply:
@matks Yeah :blush: